packages/desktop-client/src/components/modals/AccountAutocompleteModal.tsx

```diff
@@ -42,11 +42,10 @@ export function AccountAutocompleteModal({
       focusAfterClose={false}
       {...modalProps}
       onClose={_onClose}
-      padding={0}
+      padding={10}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** would it make sense to move the `padding=10` inside the `Modal` component as the default value? Seems like it's used universally almost everywhere now.

Reply:
Also `style = { flex: 1, borderRadius: '6px' }`